Tests/SessionTests.swift

```diff
@@ -1108,9 +1108,9 @@ final class SessionTestCase: BaseTestCase {
             XCTAssertTrue(session.activeRequests.isEmpty)
         }
 
-        if let error = response?.result.error?.asAFError {
-            XCTAssertTrue(error.isRequestAdaptationError)
-            XCTAssertEqual(error.underlyingError?.asAFError?.urlConvertible as? String, "/adapt/error/2")
+        if let error = response?.result.error {
```

Comment:
Just marking another instance where the error *should* be wrapped in an adapter error.

Reply:
Reverted.